Short Bio
================

I am a Biologist and M.Sc. Bioinformatics from the [Universidad Nacional
de Colombia](https://unal.edu.co/). Currently I am a PhD student at the
Department of Plant Biology at [SLU](https://www.slu.se/), working in
[Dr. Hafrén’s lab](https://andershafren.wixsite.com/website) on in
Plant-Virus Molecular interactions. My Ph.D. research focuses on how
plant viruses manipulate Arabidopsis’s post-transcriptional regulatory
mechanisms, including RNA interference machinery, Stress granules, and
Translational regulation.

## Research experience

Since the earliest steps of my education, I have been fascinated by
basic research on virology and evolutionary biology, later in my career,
I found out that the development and application of computational
methodologies is of great help to answer questions on the intersection
of these fields.

As a summary of my academic experience, during my master thesis in
Dr. Clara Bermudez’s laboratory, I conducted computational analyses on
small-RNAs dataset derived from multiple time-points of Dengue viral
infected humans cell lines. In this project, I characterized
Dengue-induce expression induction of miRNAs
\[[1](#ref-alvarez2019dengue)\] and tRNA-derived fragments -a set of
ncRNA molecules that required the development of new computational tools
for their proper characterization. Specifically, I developed a
bioinformatics tool called
[NBlockTester](https://github.com/AimerGDiaz/NBlockTester)
\[[2](#ref-gutierrez2024systematic)\] to distinguish the expression of
small RNAs from background noise in the ncRNA source.

More recently, before beginning my Ph.D. in Sweden, I had the
opportunity to work as visiting research in two laboratories. First in
the Dr. Erich Grotewold lab, I worked with transcriptomic and small
RNA-seq data to characterize a synergistic effect driven by the
interfering role of trans-acting small RNAs in the regulation of
specialized metabolism genes in *Arabidopsis thaliana*
\[[3](#ref-jiang2020synergy)\]. My second visiting research experience
was in Dr. Adami’s lab, where I investigated the evolutionary
persistence of a unique type of small ncRNAs found in *Trypanosome
brucei* mitochondria, called guide RNAs. These molecules not only play a
role in RNA editing of mRNAs but may also function as a mutational
robustness system that protects the mitochondrial genome during severe
and periodic bottlenecks.

Lastly, In collaboration with [David
Salcedo](https://github.com/drTakuOmics) at David Smith’s lab, I
co-developed the ProEnd database, which analyzes HbYX motif-containing
proteins across all life proteomes, of course including viruses
\[[4](#ref-salcedo2024proend)\]. Our work has focused on exploring the
evolutionary and functional diversity of these motifs, particularly in
relation to proteasome regulation. Through this collaboration, we have
integrated bioinformatics approaches with experimental validation,
identifying novel candidates that may expand the known roles of
HbYX-like sequences.

## Teaching activities

I’ve developed the [Sociovirology
Lab](https://github.com/AimerGDiaz/Sociovirology_Lab), a laboratory
practice in which masters students investigate how variations in viral
genetics lead to differential impacts on plant growth and defense
trade-off. This is achieved using the compatible *Nicotiana benthamiana*
/ *Potyvirus rapae* (Turnip Mosaic Virus) pathosystem.

## Public repositories

- [ProEnd](https://github.com/AimerGDiaz/proend-scripts)

- [NBlockTester](https://github.com/AimerGDiaz/NBlockTester)

- [Selection of codes in bash, awk, perl, python and
  R](https://github.com/AimerGDiaz/Codes_Repository)

# PhD projects

Private repositories

- [Vertical Virus Transmission in RNA Interference Deficient
  Plants](https://github.com/AimerGDiaz/TRoV_NextGeneration)

- [Rider of the Plant’s defense storm: Turnip Mosaic Virus modulates
  Plant’s stress granule
  response](https://github.com/AimerGDiaz/TuMV_SG_repurposing)

- [CaMV and the Nightless Day: The Virus That Keeps Arabidopsis in a
  State of Restless
  Translation](https://github.com/AimerGDiaz/UPSC_Riboseq)

- [Mutually Exclusive Binding of P6 Complexes to dsRNA or TOR Determines
  Its Role in Transactivation vs RNA-granules
  Condensation](https://github.com/AimerGDiaz/ProteinStructure)

# Additional repositories

- [IP-Proteomis: Dfifrential Viral manipulation of
  UPBs](https://github.com/AimerGDiaz/Ub_IP_TuMV-CMV-CaMV)

- [CaMV manipulation of autophagy
  pathway](https://github.com/AimerGDiaz/IP_ATG8a-NBR1)

- [Viral
  Meta-Transcriptomics](https://github.com/AimerGDiaz/PlantVirology)

- [PTM Proteomics](https://github.com/AimerGDiaz/Proteomics_PTM)

-a [RNA editing in *Trypanosomes* as mutational Robustness
mechanism](https://github.com/AimerGDiaz/Trypanosomes_RNA_editing)

- [*Trypanosomes* Guide RNAs
  Networks](https://github.com/AimerGDiaz/gRNA-Annotation)
  <!--https://medium.com/@evanca/set-up-your-portfolio-website-in-less-than-10-minutes-with-github-pages-d0efa8ff56fd-->

# Science communication

- [Bio-Viroloía](https://github.com/AimerGDiaz/Bio-Virologia)

# Scientific publications

<div id="refs" class="references csl-bib-body">

<div id="ref-alvarez2019dengue" class="csl-entry">

1\. Álvarez-Dı́az DA, Gutiérrez-Dı́az AA, Orozco-Garcı́a E, Puerta-González
A, Bermúdez-Santana CI, Gallego-Gómez JC. Dengue virus potentially
promotes migratory responses on endothelial cells by enhancing
pro-migratory soluble factors and miRNAs. Virus research.
2019;259:68–76.

</div>

<div id="ref-gutierrez2024systematic" class="csl-entry">

2\. Gutierrez-Diaz A, Hoffmann S, Gallego-Gómez JC, Bermudez-Santana CI.
Systematic computational hunting for small RNAs derived from ncRNAs
during dengue virus infection in endothelial HMEC-1 cells. Frontiers in
Bioinformatics. 2024;4:1293412.

</div>

<div id="ref-jiang2020synergy" class="csl-entry">

3\. Jiang N, Gutierrez-Diaz A, Mukundi E, Lee YS, Meyers BC, Otegui MS,
et al. Synergy between the anthocyanin and RDR6/SGS3/DCL4 siRNA pathways
expose hidden features of arabidopsis carbon metabolism. Nature
communications. 2020;11:2456.

</div>

<div id="ref-salcedo2024proend" class="csl-entry">

4\. Salcedo-Tacuma DM, Howells G, McHose C, Gutierrez-Diaz A, Smith DM.
ProEnd: A comprehensive database for identifying HbYX motif-containing
proteins across the tree of life. bioRxiv. 2024;2024–06.

</div>

</div>
